pkg/gatherers/clusterconfig/clusterconfig_gatherer.go

```diff
@@ -48,6 +48,7 @@ func failableFunc(function gathererFuncPtr) gatheringFunction {
 var gatheringFunctions = map[string]gatheringFunction{
 	"pdbs":                              failableFunc((*Gatherer).GatherPodDisruptionBudgets),
 	"metrics":                           failableFunc((*Gatherer).GatherMostRecentMetrics),
+	"dvo_metrics":                       importantFunc((*Gatherer).GatherDVOMetrics),
```

Comment:
I think we can mark this as failable. I also think we should create a task for removing this flag, because it's not necessary IMO

Reply:
Oh yeah, my bad, you're definitely right about that.